### ISSUE TITLE
Would UltraStar-Deluxe like to help standardize .lrc format?

### ISSUE THREAD

**Issue by marz1877** [UltraStar-Deluxe/USDX#607]
https://github.com/marz1877/lrc_SyncedLyrics_v2

**Comment by marz1877** [comment]
@s09bQ5 

> We need pitch and timing information for every syllable.

"pitch and time for melody" will be written to a midi file and
lyrics and it's time will be handled by .lrc file

**Comment by marz1877** [comment]
@s09bQ5

> We need pitch and timing information for every syllable.

"pitch and time for melody" will be written to a midi file and
lyrics and it's time will be handled by .lrc file

**Comment by s09bQ5** [comment]
But the examples given on that page look like the timing information in an .lrc just tells us when a phrase starts. We need to know for each syllable in a phrase when it starts and ends.

**Comment by marz1877** [comment]
> But the examples given on that page look like the timing information in an .lrc just tells us when a phrase starts. We need to know for each syllable in a phrase when it starts and ends.

I the current standard https://en.wikipedia.org/wiki/LRC_(file_format)#Simple_format_extended both start of the line and each word can be specified. For syllbal I would like you to collaborate with me to improve the draft for the standard .

> Actually to me your .lrc format looks like yet another subtitle format. There is nothing musical to it. Have you considered using [SSA/ASS](https://en.wikipedia.org/wiki/SubStation_Alpha) instead of inventing a new format? It is supported by many media players and has lots of bells and whisles.

SubStation Alpha looks like movie oriented. having position, size, font, color.
My .lrc v2 specs have meaning, translation, karaoke, chords.

**Comment by dgruss** [comment]
this issue describes neither a bug nor feature request

**Comment by barbeque-squared** [comment]
yeah I don't get what the _end goal_ of this is supposed to be either. no idea if this is meant to get some unholy combination of lrc+midi into USDX (not happening, we finally got rid of the XML support) or something else. I don't know what that _something else_ is either because it's presumably fairly trivial to generate an lrc+midi from **a single** txt file, but the other way requires **two** input files already and that creates an amount of edge cases I don't even want to think about.

it also relies on lrc providers actually making them, which I don't see happening. One usecase I know of is that they're sometimes used at festivals/concerts to sync up sing-along lyrics on massive screens, and a lot of them will not bother with separate words, or if they do, they usually just fake it by interpolating the entire sentence.

even if the end goal is to have some kind of synergy between the music industry and karaoke games, I just don't see that happening. the music industry has little incentive to do anything that doesn't directly benefit them, and for these open-source karaoke programs, I think it's better to stay as far away from it as possible.

**Comment by barbeque-squared** [comment]
I did see that Yass Reloaded [in its 2026.5.0 release](https://github.com/DoubleDee73/Yass-Reloaded/releases/tag/v2026.5) had a note about LrcLib integration. Not sure what it does or if it's even this version of .lrc, but it's probably a better place to have this kind of integration.
